.ddev/config.yaml

```diff
@@ -0,0 +1,268 @@
+name: vpge
+type: drupal9
+docroot: docroot
+php_version: "8.1"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames: []
```

Comment:
Remember, this is a multisite, with both vpge and diversityworks.  Dunno if that's relevant to your purposes, but probably worth taking into consideration.  The lando config has entries for both, but if you don't need them, then 🤷

Reply:
@imonroe Good suggestion. I added `diversityworks` url and tested it out on my local. Good to go.